test/Atmos/EDMF/closures/mixing_length.jl

```diff
@@ -91,9 +91,9 @@ function mixing_length(
     w_up = vuntuple(i -> up[i].ρaw / up[i].ρa, N_up)
     b = sum(
         ntuple(N_up) do i
-            a_up[i] * w_up[i] * δ[i] / env.a *
+            a_up[i] * w_up[i] * Δ[i]/max(up[i].ρaw,eps(FT)) / env.a *
```

Comment:
Should be: ` Δ[i] / gm.ρ/ env.a *`

Reply:
by definition  δ[I]=Δ[i]/up[i].ρaw

Reply:
I know, but you have ` a_up[i] * w_up[I]` in front, which was used because `up[i].ρaw*δ[I]=Δ[i].` If you look at equation (19) in my paper you will see that the strange I is b: The formulation is valid in term of Δ, we do not need updraft area fraction or vertical velocity in the denominator.